### DISCUSSION TITLE
richiesta informazioni - rendere smart lo scaldabagno di casa

### DISCUSSION THREAD

**Discussion by gab-chila** [indomus/forum#638] [Hardware & Firmware]
Ciao a tutti, 
Vorrei poter rendere smart il mio scaldabagno elettrico. Al momento questo si accende e spegne mediante un semplice interruttore.
Vorrei poter installare invece uno switch wifi per poterlo gestire da remoto e temporizzare

Su internet si trova molto poco. Sarei indirizzato verso questo prodotto:

‎FreeLeben - ‎R3320015U114QN_SML
link di amazon:
https://www.amazon.it/FreeLeben-Interruttori-Intelligenti-Scaldabagno-Compatibile/dp/B07T7N9LFJ

ho letto parecchie recensioni negative però (con annesse foto del prodotto arrosto) e non vorrei brutte sorprese :-)

ho visto anche questo che sembra molto famoso:
Sonoff POW R2
link di amazon:
https://www.amazon.it/POW-R2-Interruttore-Intelligente-Energetico/dp/B07C88WLM2

Sapreste indicarmi quale potrebbe essere la scelta migliore (non necessariamente tra questi due) per le mie esigenze?

in allegato trovate le foto con le specifiche del mio scaldabagno e l'interruttore (si tratta di una scatola 503, quindi se possibile vorrei dei suggerimenti anche a proposito dell'istallazione: placche, mascherine adatte, ecc. Nel caso del Sonoff ad esempio, non saprei proprio come "montarlo" in quella posizione)

Grazie per la disponibilità

![1](https://user-images.githubusercontent.com/89274351/130281164-0bc9232f-52aa-4122-8125-d856e85c7b79.jpg)
![2](https://user-images.githubusercontent.com/89274351/130281165-75ec070f-c21b-4c4c-8159-3942408dcb08.jpg)

**Comment by essegi65** [comment]
Non conosco l'interruttore del tuo primo link, stai attento alla potenza del tuo scalda bagno (non credo sia sotto i 10A) quindi ti serve uno strumento che possa reggere fino a 16A.
Io per tutti gli elettrodomestici di quella potenza ho optato per i sonoff pow o shelly 1 pm.
Entrambi ti daranno, oltre la possibilità di accendere e spegne da remoto (con il sonoff dovrai modificare il tuo impianto elettrico in maniera più importante), il controllo dei consumi.
Lo shelly 1pm è più facile da montare anche dentro una 503.
Li puoi inserire in HA facilmente entrambi con i comandi MQTT.
Io opterei per uno di quelli.
Ciao

**Reply by Maspar79** [reply]
Concordo in pieno con essegi65

**Reply by gab-chila** [reply]
Ciao Essegi65
grazie mille per le informazioni.
Non conoscevo lo Shelly 1 PM e in questo periodo mi sto un po' documentando.... ho visto che è uscito anche lo Shelly Button che farebbe anche da interruttore acceso/spento, il che farebbe al caso mio: il problema è che rimarrebbero dei fili "scoperti", quelli cioè che uscirebbero dalla 503 per andare nello Shelly.

il mio scopo sarebbe proprio quello di sostituire l'interruttore con un interruttore smart che sia si programmabile da app, ma in caso anche azionabile direttamente dal muro, è possibile che non esista un apparecchio in grado di fare entrambe le cose? (il concetto è quello dell'interruttore delle luci, solo che dovrebbe essere in grado si reggere il wattaggio e amperaggio di uno scaldabagno)

**Reply by Maspar79** [reply]
Ciao, lo shelly button non va bene, compra tranquillamente lo shelly pm, dopodiché interponi un semplice Interruttore a comando dello shelly, se cerchi in rete troverai gli schemi.

**Reply by essegi65** [reply]
Confermo. Secondo me la tua soluzione è di mettere uno shelly 1pm dentro la 503. Lo comandi con l'interruttore normale (vedi [qui](https://indomus.it/guide/integrare-shelly-1pm-a-home-assistant-via-mqtt/) fai conto di sostituire alla lampadina il tuo scaldabagno).
